### ISSUE TITLE
Get rid of tab bar

### ISSUE THREAD

**Issue by ualch9** [OneBusAway/OBAKit#267]
In my quest for the very best accessibility, one change I made is defaulting the map drawer to be full screen when the user is in voiceover. The map is difficult (in my experience) to navigate in voiceover and it is more helpful to provide a list of data instead of the map. 

A full screen drawer as-is feels bare, so to make the change more meaningful, I added bookmarks into the drawer.
This provides two benefits for both standard and accessibility users,
- User awareness of the bookmarks
- A "speed dial" for accessing bookmarks

Upon adding bookmarks, **the tab bar feels redundant**, there's no need to have a separate page for bookmarks when it is already presented to me on the "first" screen.

Today View bookmarks are shown at normal state. On that topic, I suggest renaming Today View Bookmarks to Favorite Bookmarks, for the sake of unifying Today View, Drawer "speed-dial", and Widget.

Getting rid of tab bar also means integrating Recent stops somewhere. This is up to discussion, but I think it should be kept in the focused state of the drawer because Nearby stops is more relevant when the user is focused on the map. Nearby stops stay there until the user types something in the search bar. 

To be clear, for the time-being, normal UINavigationController view presentation behavior continues.

I realize this is basically copying Maps.app, but I feel this is how we can provide a better accessibility experience and keep navigation concise for standard users.

# Mockups
## Yes Tab Bar
| Before | Tab bar + Bookmarks |
|-------|--------|
| ![Before](https://user-images.githubusercontent.com/22162410/88619887-73af0600-d051-11ea-99b3-252ad19fffa7.png) | ![Standard + Bookmarks](https://user-images.githubusercontent.com/22162410/88620504-1025d800-d053-11ea-9972-572a7c15f2cc.png)|

### Normal
| Normal State | Normal Full Screen State (Voiceover point-of-entry) |
|-----|-------|
| ![standard](https://user-images.githubusercontent.com/22162410/88620242-5a5a8980-d052-11ea-9054-adf9837612f6.png) | ![standard-full-screen](https://user-images.githubusercontent.com/22162410/88620251-5f1f3d80-d052-11ea-8fd2-3a721525860c.png) |

### Focused (search bar tapped)
| Focused State | Focused Full Screen State |
|------|-------|
| ![focused](https://user-images.githubusercontent.com/22162410/88620274-6cd4c300-d052-11ea-8556-d37a9007b6e1.png) | ![focused-full-screen](https://user-images.githubusercontent.com/22162410/88620278-6e9e8680-d052-11ea-81c4-65825dbe23e1.png) |

I am proposing a big UI change for the app. However (based purely on anecdotal evidence), I think this is a good time to make these changes. Not as many people are using public transport at the moment and pushing out a change like this will allow us to gather early feedback with a subset of users who are still using OBA before commuters return (hopefully!)

**Comment by aaronbrethorst** [comment]
I am very open to this, albeit somewhat leery of major changes to app Ux. That said, I think your observation about getting early feedback while public transit is relatively unused is astute.

@alanborning thoughts?

**Comment by alanborning** [comment]
My reaction is similar to @aaronbrethorst's -very open to this - particularly if it improves accessibility - with the concern about major changes to the Ux. @ualch9 - have you run this idea by any blind or low vision OneBusAway users?

**Comment by ualch9** [comment]
@alanborning I have not consulted anyone, nor do I "have access" to anyone blind. I think the proposed drawer implementation will work well as Apple Maps is basically the same UX and Apple is invested in accessible designs.

That being said, I did want to see how the app may look for a low-vision user. This is simulated with blurriness and reduced color saturation in the standard text size.
<sub>dont stare into the picture for too long, my eyes hurt now :(</sub>
<img width="1085" alt="Screen Shot 2020-07-28 at 2 21 03 PM" src="https://user-images.githubusercontent.com/22162410/88723996-9e936b80-d0de-11ea-8f42-14e812f5f718.png">

The icons and colors/shapes help a lot with identifying and differentiating the sections. If anyone has the resources to get some feedback, or point me in the right direction, that would be great!

**Comment by alanborning** [comment]
If we had some money for this, I would love to have an accessibility consultant on retainer -- ideally someone from the blind/low vision community, who could also help with outreach. In the absence of money (unfortunately the current state of affairs) I can ask our contacts at Sound Transit and King County Metro to put us in touch with e.g. the Citizens Accessibility Advisory Committee at ST, or the Accessible Services office at KCM. Would you be up for that?

**Comment by ualch9** [comment]
@alanborning Absolutely, that'd be awesome!
